### PR TITLE
run inspector

### PR DESCRIPTION
**Pull request by abuts**
This is incomplete solution for #1569 #1569 but it contains bunch of improvement for convenience working with `run_inspector`
1) better error reporting
2) better logging when filebacked objects are provided
3) allow filebacked objects analysis

Further investigations for issue are ongoing as user gives example of issues but these are independent and, I believe useful changes.

Apparently issue have been resolved. It is not with run_inspector but with old sqw file containing invalid run_id-s. After running recent `gen_sqw` all issues have gone so it is the solution

### CODE REVIEW

File: horace_core/sqw/@sqw/run_inspector.m [hunk 1]
```diff
@@ -91,8 +91,10 @@
 end
 nd=dimensions(obj);
 if nd<1 || nd>2
-    error('HORACE:run_inspector:invalid_argument',...
-        'Input dataset must be an sqw object that is 2d or 1d');
+    error('HORACE:run_inspector:invalid_argument',[...
+        '***  Input dataset must be an sqw object with 1d or 2d image.\n' ...
+        '     The image of the provided object have: %d dimensions.'], ...
```
Comment (cmarooney-stfc) [inline horace_core/sqw/@sqw/run_inspector.m:96]:
```suggestion
        '     The image of the provided object has: %d dimensions.'], ...
```

File: horace_core/sqw/@sqw/private/cut_accumulate_data_.m [hunk 2]
```diff
@@ -241,7 +241,7 @@
         % PageOp_sqw_join operation.
         pix_comb_info = cut_data_from_file_job.accumulate_pix(pix_comb_info, false, ...
             pix_ok, pix_indx, npix, ...
-            buf_size);
+            buf_size,ll);
```
Comment (cmarooney-stfc) [inline horace_core/sqw/@sqw/private/cut_accumulate_data_.m:244]: worth stating what 11 represents

File: horace_core/algorithms/@cut_data_from_file_job/cut_data_from_file_job.m [hunk 3]
```diff
@@ -238,6 +238,11 @@
             % npix             The npix array associated with this chunk of pixels
             % max_buf_size     The maximum buffer size for reading/writing
             % npix_retained     Number of pixels retained in this chunk of the cut
+            % Optional:
+            % log_level        verbosity of the accumulate algorithm as
+            %                  defined in hor_config.log_level. If absent, 
+            %                  hor_config.log_level will be used to
+            %                  estimate the verbosity
```
Comment (cmarooney-stfc) [inline horace_core/algorithms/@cut_data_from_file_job/cut_data_from_file_job.m:245]:
```suggestion
            %                  define the verbosity
```
can't see any estimation here

**Review by cmarooney-stfc — CHANGES_REQUESTED** [review]

File: horace_core/sqw/@sqw/private/cut_accumulate_data_.m (hunk 2 above)
Comment (abuts) [reply horace_core/sqw/@sqw/private/cut_accumulate_data_.m:244]: well, it is described and your requessting to modify the description wording in the next comment

**Pull request merged**